### PR TITLE
added two smart bee designs boards

### PR DESCRIPTION
**Pull request by strid3r21**
first time posting a PR in this repo so please let me know if im missing anything or need to change/add more.

thanks

### CODE REVIEW

File: ports/espressif/boards/smartbeedesigns_bee_motion_s3/board.h [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+// USB UF2
+//--------------------------------------------------------------------+
+
+#define USB_VID           0x303A
+#define USB_PID           0x8115
+#define USB_MANUFACTURER  "Smart Bee Designs"
+#define USB_PRODUCT       "Bee Motion S3"
+
+#define UF2_PRODUCT_NAME  USB_MANUFACTURER " " USB_PRODUCT
+#define UF2_BOARD_ID      "Bee-Motion-S3-v1.0"
```
Comment (hathach) [inline ports/espressif/boards/smartbeedesigns_bee_motion_s3/board.h:72]: Board ID must follow the syntax of https://github.com/microsoft/uf2 check other board for reference

**Review by hathach — COMMENTED** [review]
thank for the pr, please
- correct the board id
- also add your board to ci build https://github.com/adafruit/tinyuf2/blob/master/.github/workflows/build_esp32.yml

**Review by hathach — CHANGES_REQUESTED** [review]
please revert the unnecessary white space changes to the yml file.

**Comment by strid3r21** [comment]
i think its fixed. prettier was adding the additional white spaces when I saved the file.

**Review by hathach — CHANGES_REQUESTED** [review]
again, please revert the unnecessary changes to the yml file. Don't edit the whole file, you only need to add your board to the list in ABC order.

**Comment by strid3r21** [comment]
> again, please revert the unnecessary changes to the yml file. Don't edit the whole file, you only need to add your board to the list in ABC order.

What changes are you referring to? All ive done is add my two boards to the list. I thought you were originally referring to the indention being different from the master, but now I'm not sure what you're talking about.

**Comment by hathach** [comment]
> > again, please revert the unnecessary changes to the yml file. Don't edit the whole file, you only need to add your board to the list in ABC order.
> 
> What changes are you referring to? All ive done is add my two boards to the list. I thought you were originally referring to the indention being different from the master, but now I'm not sure what you're talking about.

there is the "File Changes" tab in the top of the PR, you should always double check that.

**Comment by strid3r21** [comment]
> > > again, please revert the unnecessary changes to the yml file. Don't edit the whole file, you only need to add your board to the list in ABC order.
> > 
> > What changes are you referring to? All ive done is add my two boards to the list. I thought you were originally referring to the indention being different from the master, but now I'm not sure what you're talking about.
> 
> there is the "File Changes" tab in the top of the PR, you should always double check that.

Oh I think i see what's wrong now. I'll try and fix it in a few hours once I'm home..sorry for the hassle.

**Pull request closed without merging**